changelog.dd

```diff
@@ -8,11 +8,26 @@ $(BUGSTITLE Library Changes,
 
 $(LI $(REF GCAllocator.goodAllocSize, std,experimental,allocator,gc_allocator)
 was added.)
+$(LI $(RELATIVE_LINK2 std-algorithm-iteration-scan, $(XREF algorithm.iterator,
+scan) was added.))
 
 )
 
 $(BUGSTITLE Library Changes,
 
+$(LI $(LNAME2 std-algorithm-iteration-scan, $(XREF algorithm.iterator, scan)
+was added.)
+
+    $(P $(XREF algorithm.iterator, scan) returns the successive reduced values
+        of an input range. 
+    )
+
+    ------
+    assert([1, 2, 3, 4, 5].scan!"a + b".array == [1, 3, 6, 10, 15]);
```

Comment:
Nitpick, I'd really prefer we use proper lambda functions here instead of string lambdas